src/main/java/org/opensearch/flowframework/transport/CreateWorkflowTransportAction.java

```diff
@@ -311,6 +311,7 @@ void checkMaxWorkflows(TimeValue requestTimeOut, Integer maxWorkflow, ActionList
             try (ThreadContext.StoredContext context = client.threadPool().getThreadContext().stashContext()) {
                 logger.info("Querying existing workflows to count the max");
                 client.search(searchRequest, ActionListener.wrap(searchResponse -> {
```

Comment:
(Personal preference) While we're mucking around in this bit of code, it'd be nice to change the name of the `searchRequest` to `countRequest` to more clearly indicate we've set `size(0)`.

Reply:
is `searchRequestHitCount` okay since its not a countRequest which is something different so also can add confusion https://opensearch.org/docs/latest/api-reference/count/, https://github.com/opensearch-project/OpenSearch/blob/main/client/rest-high-level/src/main/java/org/opensearch/client/core/CountRequest.java

Reply:
Let's leave it for now. I've seen `countRequest` elsewhere in the repo. RHLC actually has a CountRequest which implements it this way under the hood.  Just a future thought.